lib/rules/no-console.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview Add fixer to rule "no-console"
+ * @author Pig Fang <g-plane@hotmail.com>
+ */
+"use strict";
+
+const ruleComposer = require("eslint-rule-composer");
+const utils = require("../utils");
+
+const rule = utils.getFixableRule("no-console");
+
+rule.meta.fixable = "code";
```

Comment:
nitpick: it has been assigned here: https://github.com/aladdin-add/eslint-plugin-autofix/blob/master/lib/utils.js#L12